.github/workflows/compare_translations.py

```diff
@@ -0,0 +1,170 @@
+"""Script to encourage more efficient coding practices.
+Methodology:
+
```

Comment:
What changes did you make to this file, that's based on the `Talawa` translations check script, and why?

Reply:
sir you have asked to make this file python black, pycodestyle, pylint and pydocstyle compliant. So, I have made some changes based on the Talawa translations check script.

Reply:
1. Did you make any changes to the logic? 
1. Will this script perform differently in any given scenario than the other one and if so, why?

Reply:
Sir, as both issues are similar, therefore I have taken a reference from that script. Furthermore, this script will work as same as that one and will produce the same result as that one.
If you want I can make changes in this.